banyand/measure/aggregate/avg.go

```diff
@@ -27,23 +27,11 @@ type Avg[A, B Input, R Output] struct {
 // Avg uses type parameter A.
 func (f *Avg[A, B, R]) Combine(arguments Arguments[A, B]) error {
 	for _, arg0 := range arguments.arg0 {
-		switch arg0 := any(arg0).(type) {
-		case int64:
-			f.summation += R(arg0)
-		case float64:
-			f.summation += R(arg0)
-		default:
-			return errFieldValueType
-		}
+		f.summation += R(arg0)
```

Comment:
We could enhance the performance by following the guidelines in https://sourcegraph.com/blog/slow-to-simd. You can implement the suggested improvement methods outlined in the article. However, I wouldn't advise using the SIMD path as it's overly complex. Kindly provide the performance comparison results by submitting benchmarks for each function.